### DISCUSSION TITLE
Getting 'unexpected end of input' on vercel only.

### DISCUSSION THREAD

**Discussion by OutThisLife** [vercel/vercel#5285] [Help]
Tried flushing vercel cache w/ `-f` but no luck. This is nextjs w/ SSG. Anyone else?

![image](https://user-images.githubusercontent.com/770929/95667773-2a3e6500-0b30-11eb-8e0c-d4d36a6e47e8.png)

**Comment by mcsdevv** [comment]
Hey @OutThisLife, that's definitely unexpected - can you get in contact [here](vercel.com/support/request) so we can assist further?

**Comment by valse** [comment]
I'm having the same issue and it seems related to the webpack 5 resolution; for the moment I removed it from my `package.json`
